pyte/graphics.py

```diff
@@ -26,11 +26,14 @@
     1: "+bold",
     3: "+italics",
     4: "+underscore",
+    5: "+blink",
+    6: "+blink",
```

Comment:
Could you remove 6? TERM=linux only supports 5, I'd prefer to stay compatible with it.